rfc/0067-extended-sdk-observability-rto-v2.md

```diff
@@ -952,12 +970,49 @@ If this information is not available for any reason (generally this will only be
 
 To aid performance, the SDK is allowed to cache these fields for up to (an arbitrarily chosen) 10 minutes.  This ensures that any cluster name changes (expected to be rare) are eventually picked up.
 
+# Compatibility with older semantic conventions
+
+SDKs that implemented earlier revisions of this RFC will have implemented semantic conventions that were considered 'Experimental' by OpenTelemetry.
+
+In order to support the stable OpenTelemetry semantic conventions, SDKs should report operation latencies to the `db.client.operation.duration` metric, instead of `db.couchbase.operations`. In addition, the following attributes have been renamed, removed, or added by the migration to the stable semantic conventions.
+
+| | Experimental (v0) | Stable (v1) | Note |
+| -- | ------------ | ------ | ---- |
+| System name | db.system | db.system.name | |
+| Bucket name | db.name | db.namespace | |
+| Query statement | db.statement | db.query.text | |
+| Operation name | db.operation | db.operation.name | |
+| Error | outcome | error.type | (1) |
+| OSI transport layer | net.transport | network.transport | (2) |
+| Local hostname/IP | net.host.name | Removed | |
+| Local port | net.host.port | Removed | |
+| Server hostname/IP | net.peer.name | server.address | |
+| Server port | net.peer.port | server.port | |
+| Peer address | Did not exist | network.peer.address | |
+| Peer port | Did not exist | network.peer.port | |
+
+**Notes:**
+
+1. In v0, one of the possible values was "Success". In v1, the attribute should be omitted if there was no error.
+2. In v0, the value was "TCP.IP". In v1, it should be "tcp" instead.
+
+## Opting into the stable semantic conventions
+
+Existing implementations should not change the conventions that they emit by default. Users can opt into the new conventions either using a `observability_stable_semantic_convention_opt_in` (`String[]` type) cluster option. Support should also be added for the `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable (values are comma-separated strings) which is  recognized by many OpenTelemetry instrumentation libraries. If both are set, the cluster option has higher precedence.
```

Comment:
I think "cluster option" would be less ambiguously represented as `a ClusterOptions field`.  At the moment, on first read this seemed to be implying there would be cluster-side configurability for this.